src/main/java/tc/oc/pgm/cycle/CycleConfig.java

```diff
@@ -31,7 +31,7 @@ public Auto matchEmpty() {
   }
 
   public Auto matchEnd() {
-    return new Auto("match-end");
+    return new Auto("match-finish");
```

Comment:
Not sure if the config change is intended, probably the user-facing side should keep this as match-end, "finish" being more of an internal name